iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -14,38 +17,108 @@ public enum ErrorCode
         // BadRequest - 400
         InvalidProtocolVersion = 400001,
 
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         DeviceInvalidResultCount = 400002,
+
         InvalidOperation = 400003,
         ArgumentInvalid = 400004,
         ArgumentNull = 400005,
         IotHubFormatError = 400006,
+
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         DeviceStorageEntitySerializationError = 400007,
+
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         BlobContainerValidationError = 400008,
+
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         ImportWarningExistsError = 400009,
+
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         InvalidSchemaVersion = 400010,
+
         DeviceDefinedMultipleTimes = 400011,
+
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         DeserializationError = 400012,
+
+        /// <summary>
+        /// An error was encountered processing bulk registry operations.
+        /// </summary>
+        /// <remarks>
+        /// As this error is in the 4xx HTTP status code range, the service would have detected a problem with the job
+        /// request or user input.
+        /// </remarks>
```

Comment:
As mentioned <remarks> don't work here,

```suggestion
        /// As this error is in the 4xx HTTP status code range, the service would have detected a problem with the job
        /// request or user input.
```